src/libgit2/odb_mempack.c

```diff
@@ -132,6 +132,34 @@ static int git_mempack__dump(
 	return err;
 }
 
+int git_mempack_write_thin_pack(git_odb_backend *backend, git_packbuilder *pb)
+{
+	struct memory_packer_db *db = (struct memory_packer_db *)backend;
+	const git_oid *oid;
+	size_t iter = 0;
+	int err;
+
+	/* TODO: Implement the recency heuristics.
+	    For this it probably makes sense to only write what's referenced
+		through commits, an option I've carved out for you in the docs.
+	    wrt heuristics: ask your favorite LLM to translate https://git-scm.com/docs/pack-heuristics/en
```

Comment:
```suggestion
	    wrt heuristics: ask your favorite LLM to translate https://git-scm.com/docs/pack-heuristics
```

Reply:
Hmm wait, I thought I removed this comment because the recency heuristics are implemented at a later stage.

Reply:
Awesome; thanks. Once CI finishes, I'll merge.

Reply:
Done, and I've squashed the edits